lib/beacon/router.ex

```diff
@@ -50,9 +50,16 @@ defmodule Beacon.Router do
         {session_name, session_opts, route_opts} = o = Beacon.Router.__options__(opts)
 
         live_session session_name, session_opts do
+          get "/beacon_assets/:asset", MediaLibraryController, :show
```

Comment:
The endpoint to serve assets, each site gets its own. The trade-off is if a site is private (behind a plug or something) which could break this endpoint but if that ever happens we can provide a customization.